test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs

```diff
@@ -95,5 +177,40 @@ public void ConstructingGrpcExporterFailsWhenHttp2UnencryptedSupportIsDisabledFo
                 Assert.Null(exception);
             }
         }
+
+        private sealed class OpenTelemetryEventListener : EventListener
```

Comment:
🚀 
this is a nice improvement for all test which run in the spun up containers..

Reply:
Cool glad you like it! Very helpful to troubleshoot when things break/sanity check it is working as intended.

Reply:
Yes!. Same with this : https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs#L419

(I have implemented something similar in the past in ApplicationInsights to get SDK logs from the docker.)